templates/Gemfile.erb

```diff
@@ -16,6 +16,7 @@ gem "normalize-rails", "~> 3.0.0"
 gem "pg"
 gem "puma"
 gem "rack-canonical-host"
+gem "rack-mini-profiler", require: false
```

Comment:
Would leaving the default value of `RACK_MINI_PROFILER=0` in production be enough?